internal/printer/print-to-js.go

```diff
@@ -293,6 +293,8 @@ func render1(p *printer, n *Node, opts RenderOptions) {
 	if n.Expression {
 		if n.FirstChild != nil {
 			p.print("${")
+		} else {
+			p.print("${undefined")
```

Comment:
Nit pick:

```suggestion
			p.print("${void 0")
 ```
 
 Wouldn’t want someone redefining `undefined` for fun side-effects.

Reply:
Smart!